packages/ember-debug/lib/error.js

```diff
@@ -18,15 +7,12 @@ function ExtendBuiltin(klass) {
   @constructor
   @public
 */
-export default class EmberError extends ExtendBuiltin(Error) {
-  constructor(message) {
-    super();
 
-    if (!(this instanceof EmberError)) {
-      return new EmberError(message);
-    }
+export default EmberError;
+function EmberError(message, code) {
+  if (this instanceof EmberError) {
+    let error = Error.call(this, message, code);
```

Comment:
We aren't actually using real super (getting transpiled) also we still support an unfortunate scenario calling the constructor without new. Which ES6 classes don't support. We will need to deprecate that first, then remove in the future.